### DISCUSSION TITLE
Welcome to Aptos Developer Discussions!

### DISCUSSION THREAD

**Discussion by 0x-j** [aptos-labs/aptos-developer-discussions#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by angel-andre** [comment]
Great idea for a place where developers in the Aptos community can discuss the finer technical aspects 👍

**Reply by br4nden** [reply]
Just discovered this now, pretty good, im trying to learn MOVE

**Reply by GH14RRTrioMac** [reply]
We e-just still observing regarding about our own individual right simulation software world soul background identity experience during our own individual right simulation software world recent generation era category creating something special extra ordinary activating gaming idle leisure effort accumulated stored deposited saving that until reaching quota amount level layer stage rank position capacity making something start when if getting become approved accepted hoping having better results response request coming from valuable treasured precious golden timely moment calling instinct nature existing objective being one divine holy sacred holy existing objective being one asset liabilities legacies property right belonging benefits cycle styliformations evolutionary existing objective individual right planetary state courtesy community concern citizens individual right courtesy future generations historical informations traditional educational technological culture religious beliefs council societies benefitenors many more some of it within among us maybe of the being within as the e-active software network system format program up-dating on-line participator believer device application social media account entertainment business company industry network iconic setting feature imaginary image monitored screen backlight wishing chance granted existing objective individual right hobby desire need purpose lifestyle courtesy seeking extra lucky charm opportunity destiny choice hoping having better results response request coming from valuable treasured precious golden timely moment calling instinct nature existing objective being one divine holy sacred holy existing objective being one our own individual right simulation software world routine schedule task activity program event participation using how can getting become accessible account contact contract address internet protocol database connectivity asset technological wallet creating something special extra ordinary crypto token coin currency exchange market trading business entrepreneur commodity company industry network producing something special extra ordinary service manufacturing elemental mattered product capital investment resources consuming consumer platform benefitenors many more some of it within among us unit price percentage rate equivalent valued coded protocol systematic equivalent valued circulatory pulsating celluliting divine holy sacred holy existing objective being one divine holy sacred holy existing objective being one asset liabilities legacies property right belonging benefits cycle styliformations evolutionary existing objective individual right planetary state courtesy community concern citizens individual right courtesy future generations historical informations traditional educational technological culture religious beliefs council societies benefitenors many more some of it within among us maybe we just still observing regarding about our own individual right simulation software world creating something special extra ordinary originated existing objective being one different typical functional usage plan example same like such dramatic scenario scene coming from valuable treasured precious golden timely moment calling instinct nature existing objective being one divine holy sacred holy existing objective being one active software network system format program up-dating on-line participator believer device application activating gaming idle leisure effort accumulated stored deposited saving platform crypto token coin currency exchange market trading business entrepreneur commodity company industry network producing something special extra ordinary service manufacturing elemental mattered product capital investment resources consuming consumer benefitenors many more some of it within among us maybe that can creating something special extra ordinary coming from valuable treasured precious golden timely moment calling instinct nature existing objective being one reliable transparencized transactional dealing negotiating access getting trying that until become approved accepted hoping having better results response request coming from valuable treasured precious golden timely moment calling instinct nature existing objective being one divine holy sacred holy existing objective being one getting trying become reliable effective improvement assurance when if hoping having better results response request during our own individual right simulation software world creating something special extra ordinary social media account entertainment business company industry network producing something special extra ordinary company server vault dashboard memory network station cubicle facility synchronize programmable computerized operator agent in-charge responsible duties providing service obligation program that can creating our own individual right simulation software world creating something special extra ordinary great sustainable salary income solution lifestyle courtesy information platform getting trying making become accessible account contact contract address internet protocol database connectivity asset technological wallet creating something special extra algorithmic mineral mining energy currency deposited saving platform crypto token coin currency exchange market trading business entrepreneur commodity company industry network producing something special extra ordinary service manufacturing elemental mattered product capital investment resources consuming consumer platform benefitenors many more some of it within among us unit price percentage rate equivalent valued coded protocol systematic equivalent valued circulatory pulsating celluliting divine holy sacred holy existing objective being one asset liabilities legacies property right belonging benefits cycle styliformations evolutionary existing objective individual right planetary state courtesy community concern citizens individual right courtesy future generations historical informations traditional educational technological culture religious beliefs council societies benefitenors many more some of it within among us

**Reply by buzzlamicjihad** [reply]
wow textwall spam @0xaptosj , might want to remove this xD

**Reply by gregnazario** [reply]
I've hidden it for now

**Reply by MHPF** [reply]
Hi there I can't find my last transaction hash

**Reply by Hongquanbkct** [reply]
Dear aptos community
I am having trouble with my Pontem wallet
when I withdraw money the system shows an error: "Bad account authentication key". Error as attached file
Please help me solve this problem
![z5838812238266_17fa3adbaa483d9df21a10f712f0a362](https://github.com/user-attachments/assets/a8f01051-7b9a-4ccf-9621-e4e6875ae975)

**Comment by TiamNaseri** [comment]
OMG

**Comment by ghost** [comment]
hello aptos community.

I represent my friends to develop aptos ecosystem products entertainment industry space

**Comment by Ann4Bear** [comment]
Hello aptos community, 

I new to developing. I’m here to be able to help in any way I can. I’m a quick learner and with having a few questions here and there. I’m quick at figuring things out. I’m here to do has I’m told until I have the final approval from y’all. Thanks for all the support and opportunity.

**Reply by buzzlamicjihad** [reply]
lets not pollute the github for builders, fren

**Comment by fororo12** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

It is my honour to be here

**Reply by nichechristie** [reply]
Hey

**Comment by fdhhjhxsjs** [comment]
I wish everyone good health! Help me please! I created a wallet on the website xpence.io and found out that they were scammers. How can I return the funds now? The website xpence.io says that the blockchain is APTOS. How can I check all this and return the funds?

**Reply by fdhhjhxsjs** [reply]
My name is Slava, please help. This was my last money.

**Reply by gregnazario** [reply]
Unfortunately, these are not recoverable.  Please see https://www.reddit.com/r/Scams/comments/1duemd1/do_not_use_xpenceio_under_any_circumstances/

**Comment by MoonShiesty** [comment]
the bots are finally here 😅😅

**Reply by 0x-j** [reply]
😅

**Reply by nichechristie** [reply]
Heyy

**Comment by Nephew209-oss** [comment]
Hi my name is Eloyd,but I've never really answered to it unless your authority other then that family and friends call me nephew.And thank you for inviting me I truly am a beginner eager to learn and most might already know of me by what I  do wrong lol.

**Comment by aboy26957** [comment]
Great idea for a place where developers in the Aptos community can discuss the finer technical aspects 👍

**Comment by nichechristie** [comment]
Hey everyone

**Comment by istiakahammed1** [comment]
![Screenshot_20250130-014910](https://github.com/user-attachments/assets/14cf893e-f388-4837-b483-72df741cb62e)
![Screenshot_20250130-014822](https://github.com/user-attachments/assets/5dbcee71-8451-402c-8e57-f4bd3a5d121e)
How to get back my balance wallet is not found

**Comment by istiakahammed1** [comment]
![Screenshot_20250130-003354](https://github.com/user-attachments/assets/af1d244e-bfee-4300-b8ba-6bfa2c19ed88)

**Comment by istiakahammed1** [comment]
Automatic address changes why?

**Comment by istiakahammed1** [comment]
![Screenshot_20250130-015400](https://github.com/user-attachments/assets/7c65660f-ad90-4a6c-a586-7d36d602a8c4)
![Screenshot_20250130-014910](https://github.com/user-attachments/assets/faf1380c-d9a1-4db8-b382-cbdb367a0ab7)
![Screenshot_20250130-014822](https://github.com/user-attachments/assets/8b25cfa7-2083-4744-acaf-72e844381a9d)
All done why send worng address? TX id 0x06123041186ff1c2bf2bc6091d3a9dd46e34aa75a045ba7395ec62126cc138c4 please help?

**Reply by 0x-j** [reply]
you need to send some apt to the address, then the account will be created.

**Reply by istiakahammed1** [reply]
0xd777d247d31e8c5c46179a0aff878444885b279e212330a07a61afdc6b8d6f8e address create my wallet?

**Reply by 0x-j** [reply]
i sent 0.0001 apt to your account, it's created now and you can transfer usdt out

**Comment by istiakahammed1** [comment]
I already created my account some ATP my wallet but how to refund my
balance

**Comment by istiakahammed1** [comment]
You can send me?

**Reply by 0x-j** [reply]
i just sent to 0.0001 APT to 0xd777d247d31e8c5c46179a0aff878444885b279e212330a07a61afdc6b8d6f8e

**Comment by istiakahammed1** [comment]
That address is not my account I don't know!?

**Comment by istiakahammed1** [comment]
I don't know who address that is
0xd777d247d31e8c5c46179a0aff878444885b279e212330a07a61afdc6b8d6f8e . If you
contract wallet address who used it and refund my address
0xe25db8593d85895f1db06f43c9c774b4bd3bfc480529234c6b756482c5dd9e62 that is
my address

**Comment by keng2537** [comment]
Excited to be here! Looking forward to learning, sharing ideas, and connecting with everyone. Let’s build something great together! 💪

**Comment by Petteryz** [comment]
这是我的提币地址，0x5e94f04e93aaf16a6925908283f47d2d7fcb89f7a3b823cad3c1f16770fc5901，我操作失误，提到了0x357b0b74bc833e95a115ad22604854d6b0fca151cecd94111770e5d6ffc9dc2b，这是转账ID,0x0ae7527de4ea0b589a5cafbbcd4eb6017078e0b193f1c043e7ff24cfe960a22b,事情发生在2025.2.2，07:24:52，80.1232usdt

**Reply by nichechristie** [reply]
What is this

**Comment by Petteryz** [comment]
我从Bybit交易所提币，提到了aptos公链
地址，我想找回这笔钱

---Original---
From: "Nichole ***@***.***&gt;
Date: Sat, Mar 8, 2025 03:39 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [aptos-labs/aptos-developer-discussions] Welcome to AptosDeveloper Discussions! (Discussion #1)




 
What is this

**Comment by Nephew209-oss** [comment]
Hi I'm nephew209 and I'm the one that blu the whole SwiftKey build that's what I do wanting to learn ryte

**Comment by tushal-primevault** [comment]
I was looking forward to get the updates on slack or mail thread specific to such kind of changes so that we can make the integration changes well in advance 
update we missed : https://aptosfoundation.org/currents/migrating-to-fungible-asset-standard
is there a way so that we can get such filtered updates
